container_pipeline/workers/test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+
+import hashlib
+import json
+import logging
+import os
+
+from container_pipeline.utils import Build, get_job_name
+from container_pipeline.lib.log import load_logger
+from container_pipeline.lib.openshift import Openshift, OpenshiftError
+from container_pipeline.workers.base import BaseWorker
+
+
+class TestWorker(BaseWorker):
+    """Test Worker"""
```

Comment:
If you can add a bit more description about TesWorker - Lets make our code documentation better along with refactoring.